lambdas/process-cri-callback/src/main/java/uk/gov/di/ipv/core/processcricallback/service/CriCheckingService.java

```diff
@@ -162,10 +162,7 @@ public void validateCallbackRequest(
             throw new InvalidCriCallbackRequestException(ErrorResponse.MISSING_OAUTH_STATE);
         }
         var persistedOauthState = getPersistedOauthState(criOAuthSessionItem);
-        if (!callbackRequest
-                .getState()
-                .equals(persistedOauthState)) { // might be ipvSessionItem.getCriOAuthSessionId() !=
-            // null ? ... : null
+        if (!callbackRequest.getState().equals(persistedOauthState)) {
```

Comment:
(not important, but you have pulled out a variable for state already, so perhaps this is just `state.equals(persistedOauthState)`)